### PR TITLE
Drop sosreport_event.conf

### PR DESCRIPTION
**Pull request by ernestask**
With 773955450f3760ffa6a0e8a8ecc7b8784a2ce451 (and the corresponding
libreport changes), the sosreport generation becomes unneeded. Anyone
who finds it useful can either copy the file from us or write their own.

### CODE REVIEW

File: tests/runtests/upload-handling/runtest.sh [hunk 1]
```diff
@@ -67,8 +67,7 @@ rlJournalStart
         rlRun "ls -ld $rem_upload_dir | grep \"drwxr-x---\. [0-9]\+ root abrt \""
 
         pushd $rem_upload_dir
-        # do not check sosreport stuff
-        for elem in $(ls | egrep -v "sosreport\.log|$(hostname -s)");
+        for elem in $(ls);
```
[the remark this answers is not in the document]
Comment (ernestask) [reply tests/runtests/upload-handling/runtest.sh:70]: I think not at all, but I suppose I could add some quotes.

[the remark this answers is not in the document]
Comment (ernestask) [reply tests/runtests/upload-handling/runtest.sh:70]: https://github.com/abrt/abrt/pull/1522

[the remark this answers is not in the document]
Comment (xsuchy) [reply tests/runtests/upload-handling/runtest.sh:70]: What was the reason of `egrep -v $(hostname -s)`? Should it be really wiped too?

[the remark this answers is not in the document]
Comment (ernestask) [reply tests/runtests/upload-handling/runtest.sh:70]: 58c43988066cb981f1db5bd130416eaa7aa80c87

https://github.com/sosreport/sos/blob/1d7bab6c7ce3f78758113ca3cdf3e9fa1762df24/man/en/sos-report.1#L222

https://salsa.debian.org/meskes/hostname/-/blob/e33538493fc7023688f474ee215a9608a304d472/hostname.1#L220

It fit, so it went.

**Pull request merged**